scripts/sfpshow

```diff
@@ -647,6 +648,7 @@ def cli():
 @click.option('-n', '--namespace', default=None, help="Display interfaces for specific namespace")
 def eeprom(port, dump_dom, namespace):
     if port and multi_asic.is_multi_asic() and namespace is None:
+        load_db_config()
```

Comment:
@anamehra instead of calling every CLI option can we call it once in the sfputil/main() function ?

Reply:
I did not see it using sfputil. 'show interfaces transceiver eeprom' CLI and others land in sfshow

Reply:
@anamehra my bad. I mean sfpshow/main() function. Just before this line once https://github.com/sonic-net/sonic-utilities/blob/master/scripts/sfpshow#L741

This way you do'nt have to call load_db_config() in every place